etc/db_schema.xml

```diff
@@ -27,4 +27,17 @@
             <column name="entity_id"/>
         </constraint>
     </table>
+    <table name="balance_charges" resource="default" engine="innodb" comment="Balance Charges">
+        <column xsi:type="int" name="entity_id" unsigned="false" nullable="false" identity="true" comment="Entity Id"/>
+        <column xsi:type="int" name="charge_id" nullable="false" comment="Charge Id"/>
+        <column xsi:type="int" name="invoice_id" unsigned="true"  nullable="false" comment="Invoice Id"/>
```

Comment:
why is this nullable?

Reply:
Its nullable="false" so this is ok right?

Reply:
correct, my bad